cmd/serve/dlna/dlna.go

```diff
@@ -97,12 +97,25 @@ func newServer(f fs.Fs, opt *dlnaflags.Options) *server {
 	if friendlyName == "" {
 		friendlyName = makeDefaultFriendlyName()
 	}
+	interfaces := make([]net.Interface, 0, len(opt.InterfaceNames))
+	for _, interfaceName := range opt.InterfaceNames {
+		var err error
+		tmp, err := net.InterfaceByName(interfaceName)
+		if err != nil {
+			fs.Errorf(nil, "Failed to resolve interface name: %v", err)
+			continue
+		}
+		interfaces = append(interfaces, *tmp)
+	}
+	if len(interfaces) == 0 {
+		interfaces = listInterfaces()
```

Comment:
`listInterfaces` also contains some logic for checking whether the interfaces are actually UP, multicast etc. Should we also perform this validation for the provided interfaces or trust the user?

Reply:
What happens if the interface is DOWN for example? Does it give an error that is sensible? Or ignore it until the interface is UP? Or ignore it forever?

Reply:
Good question. In my environment the interface seems to be ignored until the interface is UP; giving the log message `<6>INFO  : DLNA server on :7879: Started SSDP on enp0s31f6` (and only starting to work when starting the interface). However, I don't know if this is the intended behavior of `anacrolix/dms` ... In my opinion checking the same flags as in `listInterfaces` would be best.

Reply:
If it starts working when the interface comes up that can be very useful if you are starting the service at boot when the interface might not be up yet. Not a real problem nowadays though with the advent of things like systemd so if you want to do the same checks then that sounds ok to me.

Reply:
Agreed! I added the checks now.